tools/transformer/src/operations.stable.json

```diff
@@ -1,214 +1,228 @@
 {
   "stableIds": [
-    "listAccessLogsByClusterName",
```

Comment:
When doing diff please ignore the first commit as I have applied alphabetical order to the list.